src/Parser/VengeanceDemonHunter/CPM_ABILITIES.js

```diff
@@ -8,63 +14,81 @@ export const SPELL_CATEGORY = {
 const CPM_ABILITIES = [
   {
     spell: SPELLS.SPIRIT_BOMB_TALENT,
-    isActive: combatant => combatant.lv90Talent === SPELLS.SPIRIT_BOMB_TALENT.id,
+    isActive: combatant => combatant.hasTalent(SPELLS.SPIRIT_BOMB_TALENT.id),
     category: SPELL_CATEGORY.ROTATIONAL,
     getCooldown: haste => 18,
     recommendedCastEfficiency: 1.0,
+    extraSuggestion: <span><br></br>This is your core healing ability. Try to refresh it even if you have just one <SpellLink id={SPELLS.SOUL_FRAGMENT.id} /> available. </span>,
   },
   {
     spell: SPELLS.IMMOLATION_AURA,
     category: SPELL_CATEGORY.ROTATIONAL,
-    getCooldown: haste => 15,
-    recommendedCastEfficiency: 0.85,
+    getCooldown: haste => 15 / (1 + haste),
+    recommendedCastEfficiency: 0.9,
+    extraSuggestion: <span><br></br>This is a great Pain filler spell. Try to always cast it on cooldown, specially when using <ItemLink id={ITEMS.KIREL_NARAK.id} details={ITEMS.KIREL_NARAK}/> legendary to trigger it&#39;s passive. </span>,
```

Comment:
`&#39;` is unnecessary, you can just use '. It's both easier to write and easier to read.